app/Command/DCE.hs

```diff
@@ -352,7 +352,9 @@ dceCommand DCEOptions {..} = do
     -- more information than is present in `CoreFn.Module`).
     for_ mods $ \m -> lift $ do
       let mn = CoreFn.moduleName m
-      copyExterns mn "cbor" <|> copyExterns mn "json"
+      copyExterns mn "cbor" <|> do
+        removeFile (dceOutputDir </> (T.unpack $ P.runModuleName mn) </> "externs.cbor")
+        copyExterns mn "json"
```

Comment:
I don't think this is right, the branch that copies `externs.json` will fail if there is no `externs.cbor`.

Reply:
If my understanding is correct, Zephyr should always write a [dummy externs.cbor file](https://github.com/coot/zephyr/blob/6034dcb72a599fc8e20c68f1636c2c37138aadfb/app/Command/DCE.hs#L345) in its output directory during [codegen](https://github.com/purescript/purescript/blob/v0.13.8/src/Language/PureScript/Make/Actions.hs#L198).

Reply:
So externs are only written using `cbor` format now, and the `json` is left here only for compatibility with a previous version, right?

Reply:
Dummy externs written by Zephyr with PureScript v0.13.8 are now .cbor files yes but the actual externs generated alongside the CoreFn representation can either be .cbor or .json files depending on the compiler version. I ran successful tests on `purescript-halogen-realworld` with both [purescript-0.13.6](https://github.com/kl0tl/purescript-halogen-realworld/tree/purescript-0.13.6-cjs-modules) and [purescript-0.13.8](https://github.com/kl0tl/purescript-halogen-realworld/tree/purescript-0.13.8-cjs-modules).

Reply:
But older version of `purescript` will never producer `cbor`, so `removeFile` will fail and the `json` will not be copied.  Maybe you tested without actually deleting the output directory (so the `cbor` files were left around).

Reply:
The externs file removed here is written during codegen when running `zephyr`, which is built with `purescript-0.13.8`, so it will always be a .cbor file.

Reply:
You can run `npm run clean && npm install && npm run bundle` on the branches I linked earlier to convince yourself (`npm run clean` removes the output and dce-output directories).

Reply:
I see, the problem is that `zephyr` allways generates the `cbor` file; maybe we shouldn't use the default `codegen`.